rules/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Rules</title>
+    <link rel="stylesheet" href="https://bitui.netlify.app/common-css/main.css">
+    <link rel="stylesheet" href="/common-css/main.css">
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div class="navbar-container">
```

Comment:
nav could be a good element to use here not even this looks not bad. Consider changing this in a later sprint.

Reply:
sure will take this into account in next release